Logstash/logstash-input-azuretopicthreadable/logstash-input-azuretopicthreadable.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name          = 'logstash-input-azuretopicthreadable'
-  s.version       = '0.9.7'
+  s.version       = '0.9.8'
```

Comment:
Just FYI, bumping up the gem version will not automatically leads to release. We have some manual process here in between. You don't have to change the version number for the gemspec next time :-)